src/QsCompiler/DocumentationParser/DocComment.cs

```diff
@@ -317,8 +319,36 @@ void ParseMapSection(IEnumerable<Block> blocks, Dictionary<string, string> accum
                     }
                 }
 
+                if (deprecated && !deprecationProcessed)
```

Comment:
So this means that right now what is written in the Deprecated section of a doc comment takes precedence over what is in the deprecated attribute? Given that the compiler behavior only looks at the latter, and I also intend to add potentially some verification of the redirection and a code action if appropriate, I suggest to make it the other way round.

Reply:
They now get merged.